app/views/layouts/application.html.erb

```diff
@@ -1,26 +1,45 @@
 <!DOCTYPE html>
 <html>
-  <title><%= CONFIG["project_shortname"] %></title>
+  <title><% if !@title.blank? %><%= @title %> | <% end %><%= CONFIG["project_shortname"] %></title>
+  <meta name="application-name" content="<%= CONFIG["project_shortname"] %>">
+  <% if !@description.blank? %>
+  <meta name="description" content="<%= @description %>">
+  <% end %>
 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <%= csrf_meta_tags %>
-  <%# favicon_link_tag %>
+
+  <!-- Favicons -->
+  <%#= favicon_link_tag "favicon.png", rel: "icon", sizes: "192x192", type: nil %>
+  <%#= favicon_link_tag "apple-touch-icon.png", rel: "apple-touch-icon", sizes: "180x180", type: nil %>
+  <!-- <meta name="msapplication-config" content="<%= browserconfig_path format: :xml %>"> -->
+  <%#= favicon_link_tag "favicon.ico", rel: "icon", type: nil %>
 
   <!-- Stylesheets -->
   <%= stylesheet_link_tag 'application', media: 'all' %>
+  <% if !@extra_css.blank? %>
+    <% Array.wrap(@extra_css).each do |css|
+      concat "  #{css}".html_safe
+    end %>
+  <% end %>
 
   <!-- Scripts -->
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => false %>
+  <% if !@extra_js.blank? %>
+    <% Array.wrap(@extra_js).each do |js|
+      concat "  #{js}".html_safe
+    end %>
+  <% end %>
 
   <body>
     <div class="header">
       <div class="container">
         <%= link_to home_path do %>
           <div class="site_title">
             <h1><%= CONFIG["project_name"] %></h1>
-            <h2>Subtitle for CDRH Elasticsearch API Sites</h2>
```

Comment:
Yeah, I'm down with that. Just didn't want a string in there we'd have to hunt down as it was.